src/app/components/NavigationBar/NavigationBar.blocks.tsx

```diff
@@ -304,7 +304,10 @@ export const NavigationBarFull: React.FC<NavigationBarFullProperties> = ({
 	const handleCloseReceiveFunds = useCallback(() => setSelectedWallet(undefined), [setSelectedWallet]);
 
 	const sendButtonClickHandler = useCallback(() => {
-		const sendTransferPath = `/profiles/${profile.id()}/send-transfer`;
+		let sendTransferPath = `/profiles/${profile.id()}/send-transfer`;
+		if (wallets.length === 1) {
+			sendTransferPath = `/profiles/${profile.id()}/wallets/${wallets[0].id()}/send-transfer`;
+		}
```

Comment:
would it be possible to have the address selection dropdown default to the first entry if there is only 1 option instead? Then we don't have to make these route adjustments in places where we link to it as the form itself will take care of it and make it less error prone 🤔